tests/e2e/test_rosacli_cluster.go

```diff
@@ -331,6 +331,70 @@ var _ = Describe("Edit cluster",
 					To(ContainSubstring(
 						"ERR: Schedule '\"5 5\"' is not a valid cron expression"))
 			})
+
+		// Commented out until bug OCM-8408 is resolved
```

Comment:
Please add the `Exclude` label instead of commenting out

Reply:
Changed